### PR TITLE
Feature login validation

### PR DESCRIPTION
**Pull request by krishnajainGKMIT**

### CODE REVIEW

File: controllers/userControllers.js [hunk 1]
```diff
@@ -19,3 +19,5 @@ async function registerUser(req, res) {
 module.exports = {
   registerUser,
 };
+
+exports.loginUser = async (req, res) => {};
```
Comment (paras2103) [inline controllers/userControllers.js]: what is this way of exporting?

File: services/user.services.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ const { responseHelper } = require("../helpers/response.helper");
 const { passHashHelper } = require("./../helpers/password.helper");
 
 const jwt = require("jsonwebtoken");
+const { payloadValidate } = require("../helpers/payloadValidationHelper");
```
Comment (paras2103) [inline services/user.services.js]: no need

File: controllers/userControllers.js (hunk 1 above)
Comment (krishnajainGKMIT) [reply controllers/userControllers.js]: chanegs implemented sir

File: services/user.services.js (hunk 2 above)
Comment (krishnajainGKMIT) [reply services/user.services.js]: chanegs implemented sir

**Pull request merged**